### PR TITLE
Removed default priority, and set enablePodPriority true

### PR DESCRIPTION
**Pull request by yangzb09**
https://dashbase.atlassian.net/browse/DB-63
remove default priority from the templates so when priority class is enabled, they do not automatically have a priority

### CODE REVIEW

File: dashbase/values.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ license: ""
 
 https: false
 
-enablePodPriority: false
+enablePodPriority: true
```
Comment (sincejune) [inline dashbase/values.yaml]: Could you verify `storageClassName` is available in `1.8`?

File: dashbase/templates/_helpers.tpl [hunk 2]
```diff
@@ -154,9 +154,6 @@ spec:
         {{ $name }}: {{ $value | quote }}
         {{- end }}
     spec:
-    {{- if .enable_pod_priority }}
-      priorityClassName: {{ .priorityClassName | default (.default.priorityClassName | default "dashbase-low-priority") }}
```
Comment (sincejune) [inline dashbase/templates/_helpers.tpl]: Could you specify `priorityClassName` manually without this?

Comment (yangzb09) [reply dashbase/templates/_helpers.tpl]: No, I'm setting it back.

**Comment by yangzb09** [comment]
I think it can be merged, now.

File: dashbase/values.yaml (hunk 1 above)
Comment (yangzb09) [reply dashbase/values.yaml]: After patch with `--feature-gates=PodPriority=true --extra-config=apiserver.runtime-config=scheduling.k8s.io/v1alpha1=true`
All version CI work.

**Comment by khou** [comment]
i dont think we should say we support a version of kubernetes where this is an alpha feature

**Comment by yangzb09** [comment]
It throws an error in 1.10, while we use conditional
`{{- if .Capabilities.APIVersions.Has "scheduling.k8s.io/v1alpha1" }}`
> unable to recognize "STDIN": no matches for kind "PriorityClass" in version "scheduling.k8s.io/v1beta1"
unable to recognize "STDIN": no matches for kind "PriorityClass" in version "scheduling.k8s.io/v1beta1"
unable to recognize "STDIN": no matches for kind "PriorityClass" in version "scheduling.k8s.io/v1beta1"

rollback.

**Comment by yangzb09** [comment]
After discussing with @sincejune , we set enablePodPriority back to "false" in k8s values.yaml

**Comment by sincejune** [comment]
> i dont think we should say we support a version of Kubernetes where this is an alpha feature

Actually, I think it's ok to have this.
But when firstly deploying, we should highly recommend customers to use Kubernetes `>=1.11.0`.

**Pull request merged**